tests/MongoDB.Driver.Core.Tests/Core/Clusters/ClusterTests.cs

```diff
@@ -481,6 +480,60 @@ public void DescriptionChanged_should_be_raised_when_the_description_changes()
             _capturedEvents.Any().Should().BeFalse();
         }
 
+        [Theory]
+        [ParameterAttributeData]
+        public void SelectServer_should_call_custom_selector(
+            [Values(true, false)] bool withEligibleServers,
+            [Values(true, false)] bool async)
+        {
+            int numberOfCustomServerSelectorCalls = 0;
+            var customServerSelector = new DelegateServerSelector((c, s) =>
+            {
+                numberOfCustomServerSelectorCalls++;
+
```

Comment:
I would say it also unnecessary blank line

Reply:
Done.